### ISSUE TITLE
0.51 Database update fail

### ISSUE THREAD

**Issue by masterz666** [ebshimizu/stats-of-the-storm#133]
I just updated to 0.51, had to restart twice because the UI would not show up. 
Now on launch the massage says 'updating DB version 2 to version 3'. 
The number of matches decrease then stop at '520 matches left'. 

Tried 4 times, it always freezes at 520 matches left. 

OS: windows 10

**Comment by ebshimizu** [comment]
hmm I'll take a look. If you can upload your database files I can take a look and probably fix it up for you.

**Comment by ebshimizu** [comment]
actually if it decreases then stops, I'll definitely need your database since that indicates an exception happened during processing.

**Comment by masterz666** [comment]
Here are the DB files


[stats-of-the-storm.zip](https://github.com/ebshimizu/stats-of-the-storm/files/1832680/stats-of-the-storm.zip)

**Comment by ebshimizu** [comment]
Yeah it's just a function throwing exceptions. I've addressed this specific issue in 789bcd1, but since this seems isolated for now I'm going to push that update later (probably when the PTR patch goes live).

I've fixed up your database for the most recent version of the app as well:
[stats-db-masterz666-v3.zip](https://github.com/ebshimizu/stats-of-the-storm/files/1834909/stats-db-masterz666-v3.zip). Make sure you copy over all 4 files and it should load up just fine.
